Sources/Operators.swift

```diff
@@ -204,6 +245,18 @@ public prefix func -(offset: UIOffset) -> UIOffset {
     return UIOffset(horizontal: -offset.horizontal, vertical: -offset.vertical)
 }
 
+public func +(lhs: CGSize, rhs: UIOffset) -> CGSize {
+    return CGSize(width: lhs.width + rhs.horizontal, height: lhs.height + rhs.vertical)
+}
+
+public func +(lhs: UIOffset, rhs: CGSize) -> CGSize {
+    return CGSize(width: rhs.width + lhs.horizontal, height: rhs.height + lhs.vertical)
+}
+
+public func -(lhs: CGSize, rhs: UIOffset) -> CGSize {
+    return CGSize(width: lhs.width - rhs.horizontal, height: lhs.height - rhs.vertical)
+}
+
```

Comment:
I'm leaning towards leaving it as-is for now to avoid confusion – it's always easy to add features, but much harder to remove them.

I'm also tempted to remove `-` altogether and just make people use `+ -5.horizontalOffset` to make the math explicit. Thoughts?